components/atoms/Toast/toast.tsx

```diff
@@ -0,0 +1,102 @@
+import React from "react";
+
+import * as ToastPrimitives from "@radix-ui/react-toast";
+import clsx from "clsx";
+import { IoClose } from "react-icons/io5";
+
+const ToastProvider = ToastPrimitives.Provider;
+
+const ToastViewport = React.forwardRef<
+  React.ElementRef<typeof ToastPrimitives.Viewport>,
+  React.ComponentPropsWithoutRef<typeof ToastPrimitives.Viewport>
+>(({ className, ...props }, ref) => (
+  <ToastPrimitives.Viewport
+    ref={ref}
+    className={clsx(
+      "fixed top-96 z-[100] flex max-h-screen w-full flex-col-reverse p-4 sm:top-auto left-[50%] right-[50%] -translate-x-[50%] sm:flex-col md:max-w-[420px]",
+      className
+    )}
+    {...props}
+  />
+));
+ToastViewport.displayName = ToastPrimitives.Viewport.displayName;
+
+const Toast = React.forwardRef<
+  React.ElementRef<typeof ToastPrimitives.Root>,
+  React.ComponentPropsWithoutRef<typeof ToastPrimitives.Root> &{variant? : "success" | "warning" | "danger"  }
```

Comment:
`variant` being optional, does that mean there's a fourth option which isn't success, warning or danger?

Reply:
Yes... With no variant, you get a plain toast without the icon at the left 👍